WordPress/Classes/ViewRelated/Blog/BlogSelectorViewController.m

```diff
@@ -378,4 +498,49 @@ - (NSArray *)sortDescriptorsWithAccountKeyPath
     return descriptors;
 }
 
+#pragma mark - UISearchController
+
+- (void)updateSearchResultsForSearchController:(UISearchController *)searchController
+{
+    self.resultsController.fetchRequest.predicate = [self fetchRequestPredicate];
+
+    NSError *error = nil;
+    if (![self.resultsController performFetch:&error]) {
+        DDLogError(@"Couldn't fetch sites: %@", [error localizedDescription]);
+    }
+
+    [self.tableView reloadData];
+}
+
+// Improves the appearance of the arrow when displayed in a popover, by matching
+// its color to the topmost view in the scrollview
+- (void)scrollViewDidScroll:(UIScrollView *)scrollView
```

Comment:
As discussed, I'm going to push this as-is for now. I'd rather not hard code assumptions about the position of the text field in the search bar (there's no way to access it directly), and I feel that at least doing _something_ is better than the default. :)